src/medCore/process/medRunnableProcess.cpp

```diff
@@ -52,11 +52,17 @@ dtkAbstractProcess * medRunnableProcess::getProcess()
 
 void medRunnableProcess::internalRun()
 {
-    if (d->process) {
-        if (d->process->update() == 0)
+    if (d->process)
+    {
+        int res = d->process->update();
+
+        if (res == 0)
```

Comment:
Ok, I understand why you saved the output in ```res```(didn't see the following code). Forget that part.